man/clrtrust.1.md

```diff
@@ -0,0 +1,175 @@
+re-generating the trust store.
+
+Certificates can be provided by the operating system for out-of-box
+functionality. Certificates can also be provided and modified by privileged
+users.
+
+It is up to each application to make use of the trust store generated by
+`clrtrust`.
+
+## OPTIONS
+
+```
+Usage: clrtrust [-v|--verbose] [-h|--help] [-c|--internal-rehash] <command> [options]
+
+    -v | --verbose          Shows more details about execution
+    -c | --internal-rehash  Forces use of internal implementation of c_rehash
+    -h | --help             Prints this message
+
+    Commands
+        generate    generates the trust store
+        list        list CAs
+        add         add trust to a CA
+        remove      remove trust to a CA
+        restore     restore trust to previously removed CA
+        check       sanity/consistency check of the trust store
+
+clrtrust <command> --help to get help on specific command.
+```
+
+**Commands that modify the trust store require root privileges.**
+
+- `clrtrust generate [-f|--force]`
+
+  The `generate` command has no arguments and generates a unified trust store
+  composed of system-provided and user-provided certificates, if any. The
+  optional `--force` parameter will forcibly generate the trust store, even if
+  it results in an empty store. See the FILES section for paths used for trust
+  store generation.
+
+
+- `clrtrust list`
+
+  The `list` command has no arguments and outputs a list of trusted certificates
+  with the following fields:
+
+  `id` uniquely identifies the certificate. It can be used as input to other
+  `clrtrust` commands such as  `remove` or `restore`.
+
+  `File`  contains the file path of the certificate in the trust store. 
+
+  `Authority` shows the name of the organization that issued the certificate.
+    This field is extracted from the certificate file.
+
+  `Expires` shows the expiration date of the certificate. This field is
+    extracted from the certificate file.
+
+
+
+- `clrtust add [<certificateFile|Id> ...] [-f|--force]` 
+
+  The `add` command takes one or more certificates as required argument(s). The
+  certificate is identified by a file path or `Id`. The certificate file(s)
+  must be PEM-encoded with only one certificate per file. The optional
+  `--force` parameter will forcibly add the certificate to the trust store,
+  even if it is not a root CA. 
+
+  Adding a root CA to the trust store allows applications using the trust store
+  to trust the root CA certificate, trust certificate chains issued by the
+  authority, verify the authenticity of peer's certificate, and establish a
+  connection.
+  
+
+
+- `clrtrust remove [<certificateFile|Id> ...]`
+
+  The `remove` command takes one or more certificates as required argument(s).
+  The certificate is identified by a file path or `Id`. The argument can be an
+  `id` of the certificate (see the `list` command) or the file path of the
```

Comment:
Nit: `id` and `Id` are used interchangeably. Specially since i uppercase is easily confused with l lowercase, I'd rather prefer using `id` (lowercase) throughout.